packages/scraper/src/GitHub.ts

```diff
@@ -0,0 +1,173 @@
+		{
+			method: 'GET',
+			headers: {
+				Authorization: `token ${githubToken}`,
+			},
+		}
+	)
+
+	const dataFolderObj = await dataFolderResp.json()
+
+	const policyFolderGitUrl = dataFolderObj.find(
+		(folder: { name: string }) => folder.name === 'policies'
+	).git_url
+	const loadPolicyFolderResponse = await fetch(
+		`${policyFolderGitUrl}?recursive=true`,
+		{
+			method: 'GET',
+			headers: {
+				Authorization: `token ${githubToken}`,
+			},
+		}
+	)
+	const policyFolderData = await loadPolicyFolderResponse.json()
+
+	const gitFiles = policyFolderData.tree.filter(
+		(file: any) => file.path.toLowerCase().indexOf('vaccination.json') !== -1
+	)
+
+	for (let i = 0; i < gitFiles.length; i++) {
+		const file = gitFiles[i]
+		const fileData = JSON.parse(b64_to_utf8(await getContent(file.url)))
+		const rootLocation = file.path.split('/')[0]
+
+		if (fileData.links?.eligibility != null) {
+			if (fileData.links.eligibility.scrape !== false) {
+				linkList.push(
+					Object.assign(fileData.links.eligibility, {
+						RootLocation: rootLocation,
+					}) as ScrapeLink
+				)
+			}
+		} else if (fileData.links?.info != null) {
+			if (fileData.links.info.scrape !== false) {
+				linkList.push(
+					Object.assign(fileData.links.info, {
+						RootLocation: rootLocation,
+					}) as ScrapeLink
+				)
+			}
+		}
+
+		if (fileData.links?.eligibility_plan != null) {
+			if (fileData.links.eligibility_plan.scrape !== false) {
+				linkList.push(
+					Object.assign(fileData.links.eligibility_plan, {
+						RootLocation: rootLocation,
+					}) as ScrapeLink
+				)
+			}
+		}
+	}
+
+	return linkList
+}
+
+export async function createIssues(linkIssues: any): Promise<void> {
+	console.log('creating github issues for integrity mismatches')
+
+	// Specifically using a for here since array.forEach + async calls
+	// can cause issues.
+	for (const location of Object.keys(linkIssues)) {
+		const titleLabel = `Scraped changes for ${location}`
+		let bodyText = ''
+		linkIssues[location].forEach((link: ScrapeLink) => {
+			bodyText += link.url + '\n'
+		})
+
+		await fetch(
+			`https://api.github.com/repos/${githubRepoOwner}/${githubRepoName}/issues`,
+			{
+				method: 'POST',
+				headers: {
+					Authorization: `token ${githubToken}`,
+				},
+				body: JSON.stringify({
+					title: titleLabel,
+					body: bodyText,
+					labels: ['scrapped changes'],
+				}),
+			}
+		)
+	}
+}
+
+const getContent = async (url: string) => {
+	const response = await fetch(`${url}`, {
+		method: 'GET',
+		headers: {
+			Authorization: `token ${githubToken}`,
+		},
+	})
+	const data = await response.json()
+
+	return data.content
+}
+
+function atobLookup(chr: any) {
+	const keystr =
+		'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/'
+	const index = keystr.indexOf(chr)
+	return index
+}
+
+function b64_to_utf8(str: string) {
+	const blob = atob(str) || ''
+	return decodeURIComponent(escape(blob))
+}
+
+function atob(data: string) {
```

Comment:
Can you add a comment header here to indicate intent of the bitmunching?

Reply:
I dropped a comment on the previous method but also swapped this bit fun for a yarn package that seems a little more straight forward.